packages/make-styles/src/makeStyles.test.ts

```diff
@@ -194,28 +194,8 @@ describe('makeStyles', () => {
     // Style elements should be different for different renderers
     expect(rendererA.styleElements.d).not.toBe(rendererB.styleElements.d);
 
-    expect(rendererA).toMatchInlineSnapshot(`
-      .f22iagw {
-        display: flex;
-      }
-      .frdkuqy {
-        padding-left: 10px;
-      }
-      .f81rol6 {
-        padding-right: 10px;
-      }
-    `);
-    expect(rendererB).toMatchInlineSnapshot(`
-      .f22iagw {
-        display: flex;
-      }
-      .frdkuqy {
-        padding-left: 10px;
-      }
-      .f81rol6 {
-        padding-right: 10px;
-      }
-    `);
+    expect(rendererA).toMatchInlineSnapshot(``);
```

Comment:
not sure if this is correct. can you pls advise @layershifter ?

Reply:
Yes, it's incorrect, snapshot should contain rendered styles.

Reply:
can you please help us resolve the problem ?

Reply:
As I see in 23ac14c you solved, the problem is described in jsdom/jsdom#3179. Removal of `createFakeDocument()` sounds reasonable to me 👍

Reply:
I already solved in the morning. thanks anyways

Reply:
if you're removing fake doucments please add a cleanup phase like in 9b3a49b

since the renderer will keep appending style tags between each test